libs/fluid-elements/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "rootDir": ".",
+    "baseUrl": ".",
```

Comment:
This will disallow importing from other libraries I would remove it.

please try to extend from a root tsconfig that has the path aliases and the `rootDir` and `baseUrl` set. Otherwise we cannot import elements form other elements.

The `emitDecoratorMetadata`, `experimentalDecorators`, `module`,  `moduleResolution` and `sourceMap` should be in this root tsconfig as well.